translations/DE.ini

```diff
@@ -2141,10 +2141,10 @@ SCRIPT_EXIT_DESC="Schließt alle Sitzungen und beendet das Programm."
 SCRIPT_OPEN_DESC="Verbindet zum Server."
 SCRIPT_CLOSE_DESC="Schließt die Sitzung."
 SCRIPT_SESSION_DESC="Zeigt alle verbundenen Sitzungen oder wählt die aktive Sitzung aus."
-SCRIPT_PWD_DESC="Zeigt den Name das entfernte Arbeitsverzeichnisses an."
+SCRIPT_PWD_DESC="Zeigt den Namen des entfernten Arbeitsverzeichnisses an."
 SCRIPT_CD_DESC="Wechselt das entfernte Arbeitsverzeichnis."
 SCRIPT_LS_DESC="Zeigt den Inhalt des entfernten Ordners an."
-SCRIPT_LPWD_DESC="Zeigt den Name des lokalen Arbeitsverzeichnisses an."
+SCRIPT_LPWD_DESC="Zeigt den Namen<< des lokalen Arbeitsverzeichnisses an."

```

Comment:
Thanks for the corrections – The `<<` here looks like a typo. Is it?

Reply:
yes it is

Reply:
i corrected it.